hw/ip/otbn/util/otbn-as

```diff
@@ -1040,6 +1040,11 @@ class Transformer:
         pos = 0
         self.line_number += 1
 
+        # Append a newline if the line doesn't have one. In practice, this just
+        # happens with the last line of a file that doesn't end with a newline.
+        if line and line[-1] != '\n':
+            line = line + '\n'
```

Comment:
Hmm, is our expectation that the assembler can modify the input file? Wouldn't it better to just throw a warning?